File: NewDependencyDemo/app/src/main/java/ca/six/dep/MainActivity.java
package ca.six.dep;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import ca.six.biz.CustomVersion;
import ca.six.common.VersionHelper;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        String v2 = new CustomVersion().getFormatVersion();
        int v1 = new VersionHelper().getVersion();
    }
}

File: IO2018/app/src/main/java/ca/six/io2018/biz/main/MainActivity.kt
package ca.six.io2018.biz.main

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.Navigation
import ca.six.io2018.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    override fun onSupportNavigateUp()
        = Navigation.findNavController(this, R.id.fragmentInMain)
            .navigateUp()

}
